app/src/main/java/de/danoeh/antennapod/fragment/ChaptersFragment.java

```diff
@@ -63,6 +79,10 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         emptyView.setIcon(R.drawable.ic_bookmark);
         emptyView.setTitle(R.string.no_chapters_head_label);
         emptyView.setMessage(R.string.no_chapters_label);
+        
```

Comment:
I would maybe only show the EmptyView if it is actually empty. Otherwise, you see the text that the episode has no chapters while it is loading.

Reply:
that shouldnt happen, if its empty the chapter button isnt shown. This should be a loading screen.

Reply:
It could happen when you open the chapters screen and wait for the next episode to play without closing it.

Reply:
good point, but i think then the dialog should just be dissmised, no?

Reply:
Would be a possibility, too, yeah :) I don't know if that is easily possible (keeping a reference to a fragment is not good because the Android system can recreate them). Maybe it can be dismissed based on the TAG.

Reply:
dismissed in ChaptersFragment.onMediaChanged works nicely :)